common/models/email.js

```diff
@@ -15,9 +15,10 @@
  * @inherits {Model}
  */
 
-var g = require('../../lib/globalize');
+ 'use strict';
+ var g = require('../../lib/globalize');
```

Comment:
Hmm, I think the order of jsdoc, use-strict, require() and module.exports is wrong here. The correct order is:

```js
'use strict';

var g = require('../../lib/globalize');

/** 
 * Email model. etc.
 */

module.exports = function(Email) {
  // ...
};
```